Tribler/community/tunnel/tunnel_community.py

```diff
@@ -109,10 +111,11 @@ def on_timeout(self):
         pass
 
 
-class TunnelExitSocket(DatagramProtocol):
+class TunnelExitSocket(DatagramProtocol, TaskManager):
 
     def __init__(self, circuit_id, community, sock_addr, mid=None):
         self.tunnel_logger = logging.getLogger('TunnelLogger')
```

Comment:
In a future PR this class' logger should be refactored to be named/created the same way than the rest of Tribler's codebase.